CONTRIBUTING.md

```diff
@@ -204,16 +192,15 @@ Contributions in the form of setup guides for integrations with external applica
 ## Styleguides
 ### Formatting

```

Comment:
Maybe this should be in the UI README? Also there are different style guides for diagnostic ui, setup-wizard, and patch-db-client. Here we could say, "Each component of EmabssyOS contains its own style guide."